base/algorithms.jl

```diff
@@ -0,0 +1,119 @@
+module Algorithms
```

Comment:
while reorganizing some of base into modules is a good idea, I don't see why these two functions make sense as the only thing in an Algorithms module